### ISSUE TITLE
CPS1 Original-filter comparison with PCB

### ISSUE THREAD

**Issue by paulb-nl** [jotego/jtcps1#81]
Here is a video recorded from jtcps1 20210201 with filter enabled.
https://youtu.be/HNkTJRo_ZR0

Here are videos of the pcb:
https://youtu.be/4C1vcIH7FTA

https://youtu.be/EDXE0SsaCiM

Doesn't it sound like the filter is removing too much high frequency compared to the pcb? The cymbal and drum sound very muffled.

**Comment by jotego** [comment]
I agree. I have just double check both the analogue (Forgotten Worlds schematics) and digital (JTCPS) filters in simulation, and they actually match.
I have a SF2 board too, I will compare the component values in that board. Maybe Forgotten Worlds, being the 1st game, had different values.
For now, I'd rather not change the filter. It may be wrong for SF2, but it may be right for other titles.
